src/test/kotlin/org/jetbrains/research/ml/coding/assistant/tree/PsiTreeConverterTest.kt

```diff
@@ -13,31 +13,35 @@ import org.jetbrains.research.ml.coding.assistant.util.PsiTestUtil.equalTreeStru
 import org.junit.Test
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
-import util.PsiUtil.preOrderNumbering
 import java.io.File
 
 @RunWith(Parameterized::class)
-class TreeConverterTest : ParametrizedBaseTest(getResourcesRootPath(::TreeConverterTest)) {
+class PsiTreeConverterTest : ParametrizedBaseTest(getResourcesRootPath(::PsiTreeConverterTest)) {
 
     companion object {
         @JvmStatic
-        @Parameterized.Parameters(name = "{index}: ({0})")
+        @Parameterized.Parameters(name = "{index}: ({0}, {1})")
         // TODO: which tests cases should I have?
-        fun getTestData(): List<Array<File>> = getNestedFiles(getResourcesRootPath(::TreeConverterTest)).map {
-            arrayOf(it)
-        }.toList()
+        fun getTestData(): List<Array<Any>> {
+            val files = getNestedFiles(getResourcesRootPath(::PsiTreeConverterTest))
+            val numberings = listOf(PreOrderNumbering, PostOrderNumbering)
+            return files.flatMap { f -> numberings.map { n -> arrayOf(f, n) } }.toList()
+        }
     }
 
     @JvmField
     @Parameterized.Parameter(0)
     var inFile: File? = null
 
+    @JvmField
+    @Parameterized.Parameter(1)
+    var numbering: Numbering? = null
+
     @Test
     fun `converting PSI to GumTree tree test`() {
         val inFilePsi = myFixture.configureByFile(inFile!!.name)
         val inContext = ApplicationManager.getApplication().runReadAction<TreeContext> {
-            inFilePsi.preOrderNumbering()
-            TreeConverter.convertTree(inFilePsi)
+            PsiTreeConverter.convertTree(inFilePsi, numbering!!)
```

Comment:
I can not comment on the 46 line, but maybe should we send the tree order to the function? Because now it compares two trees according to preOrder traversal. What do you think? Or is it not necessary? What do you think?